app/controllers/sections/consignee/ConsigneeIndexController.scala

```diff
@@ -0,0 +1,121 @@
+            } else if (shouldStartFlowFromConsigneeExportUkEu(ur.userTypeFromErn, destinationTypePageAnswer)) {
+              Future.successful(Redirect(controllers.sections.consignee.routes.ConsigneeExportController.onPageLoad(ern, lrn, NormalMode)))
+            } else if (shouldStartFlowFromConsigneeExcise(ur.userTypeFromErn, destinationTypePageAnswer)) {
+              Future.successful(Redirect(controllers.sections.consignee.routes.ConsigneeExciseController.onPageLoad(ern, lrn, NormalMode)))
+            } else {
+              logger.info(s"[onPageLoad] Combination of UserType ${ur.userTypeFromErn} and" +
+                s" DestinationTypePage answer $destinationTypePageAnswer not allowed on Consignee flow")
+              // TODO: Change redirect location when tasklist is built
+              Future.successful(Redirect(testOnly.controllers.routes.UnderConstructionController.onPageLoad()))
+            }
+        }
+
+    }
+  }
+
+  private def shouldStartFlowFromConsigneeExportUkEu(
+                                                      userTypeFromErn: UserType,
+                                                      destinationTypePageAnswer: MovementScenario
+                                                    ): Boolean = {
+    val gbwkAndSpecificDestinationTypes: Boolean =
+      userTypeFromErn == GreatBritainWarehouseKeeper && destinationTypePageAnswer == ExportWithCustomsDeclarationLodgedInTheUk
+
+    val xiwkAndSpecificDestinationTypes: Boolean = {
+      val validDestinationTypes: Seq[MovementScenario] = Seq(
+        RegisteredConsignee,
+        ExportWithCustomsDeclarationLodgedInTheUk,
+        ExportWithCustomsDeclarationLodgedInTheEu
+      )
+      userTypeFromErn == NorthernIrelandWarehouseKeeper && validDestinationTypes.contains(destinationTypePageAnswer)
+    }
+
+    gbwkAndSpecificDestinationTypes || xiwkAndSpecificDestinationTypes
+  }
+
+  private def shouldStartFlowFromConsigneeExcise(
+                                                  userTypeFromErn: UserType,
+                                                  destinationTypePageAnswer: MovementScenario
+                                                ): Boolean = {
+    val validDestinationTypesRegardlessOfUserTypes: Boolean =
+      Seq(
+        GbTaxWarehouse,
+        EuTaxWarehouse,
+        DirectDelivery
+      ).contains(destinationTypePageAnswer)
+
+    val gbrcAndValidDestinationType: Boolean =
+      userTypeFromErn == GreatBritainRegisteredConsignor && destinationTypePageAnswer == ExportWithCustomsDeclarationLodgedInTheUk
+
+    val xircAndValidDestinationType: Boolean = {
+      val validDestinationTypes: Seq[MovementScenario] = Seq(
+        RegisteredConsignee,
+        TemporaryRegisteredConsignee,
+        ExportWithCustomsDeclarationLodgedInTheUk,
+        ExportWithCustomsDeclarationLodgedInTheEu
+      )
+
+      userTypeFromErn == NorthernIrelandRegisteredConsignor && validDestinationTypes.contains(destinationTypePageAnswer)
```

Comment:
I think this condition says the `userType` can also be `GreatBritainRegisteredConsignor` 
![image](https://github.com/hmrc/emcs-tfe-create-movement-frontend/assets/31283663/0b501739-8f75-49e3-aac5-73bbb2d5150f)